enterprise/server/auth/auth.go

```diff
@@ -117,6 +120,10 @@ func (c *Claims) GetAllowedGroups() []string {
 	return c.AllowedGroups
 }
 
+func (c *Claims) GetGroupRoles() []uint32 {
```

Comment:
Can we combine the group ID & role into a single struct instead of returning them separately? I don't think the user of this API should have to lookup this data in two slices. 

Same for the Claims struct, is there a reason not to store the per-group data (ID + role) together? We could introduce a new field and remove the old allowedgroups field later once it's no longer used.

Reply:
Done